melissa/brain.py

```diff
@@ -25,32 +26,32 @@ def query(text):
 
     # Insert expression words in to SQLite. A word may appear more
     # than once in the expression. Only the first occurence will
-    # be kept. 
+    # be kept.
     actions_db.cur.executemany("INSERT OR IGNORE INTO expression "
-                             "values (?,?)", zlist)
+                               "values (?,?)", zlist)
     actions_db.con.commit()
 
     # Match multiple-word groups against the expression words.
     sql = "SELECT e.word, e.word_order, "\
-         +"  g.word_group, g.word_count, g.function "\
-         +"FROM expression e "\
-         +"JOIN words w "\
-         +"ON w.word = e.word "\
-         +"JOIN word_groups g "\
-         +"ON g.word_group = w.word_group "\
-         +"WHERE g.word_count > 1 "\
-         +"ORDER BY e.word_order, g.word_group"
+        + "  g.word_group, g.word_count, g.function "\
+        + "FROM expression e "\
+        + "JOIN words w "\
+        + "ON w.word = e.word "\
+        + "JOIN word_groups g "\
+        + "ON g.word_group = w.word_group "\
+        + "WHERE g.word_count > 1 "\
+        + "ORDER BY e.word_order, g.word_group"
 
     actions_db.cur.execute(sql)
     rows = actions_db.cur.fetchall()
 
     # Make the following easier to read using namedtuple.
-    sql_row_def = namedtuple('sql_row_def',
-        'word, word_order, word_group, word_count, function')
+    sql_row_def = namedtuple('sql_row_def', 'word', 'word_order', 'word_group',
+                             'word_count', 'function')
```

Comment:
I'll check, but I only accept 1 commit per PR. Please squash them to a single commit.